<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SISTEMA GOTY</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(94, 255, 228);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Nombre del Juego</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nombreAgregar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>calificacion</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>WaitCursor</cursorShape>
    </property>
    <property name="text">
     <string>Jugabilidad</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Historia</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Graficos</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="jugabilidadAgregar">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="historiaAgregar">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="graficosAgregar">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>-10</y>
      <width>21</width>
      <height>271</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Promedio</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultado">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextInteractionFlag::TextEditable</set>
    </property>
   </widget>
   <widget class="QPushButton" name="AgregarButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nombre del juego agregado</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultadoNombre">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="problemas">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>527</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
